src/autoscaler/eventgenerator/generator/evaluator.go

```diff
@@ -70,6 +70,10 @@ func (e *Evaluator) doEvaluate(triggerArray []*model.Trigger) {
 			return
 		}
 		for _, appMetric := range appMetricList {
+			if appMetric.Value < 0 {
+				e.logger.Info("should not send trigger alarm to scaling engine because there is nil-value metric", lager.Data{"trigger": trigger, "appMetric": appMetric})
```

Comment:
Again, as an admin, if I saw this message, what do I do?
The message also says there is a nil-value but in reality it is just < 0.